testData/roamingData.js

```diff
@@ -1,4 +1,5 @@
-export const roamingUrl = 'https://www.mts.by/services/roaming/world-travel/'
+export const servicesUrl = 'https://www.mts.by/services/roaming/world-travel/'
 export const tariffNameOne = '1 ГБ за границей'
 export const tariffNameTwo = 'Безлимитный интернет в роуминге'
 export const tariffNameThree = 'Открытый интернет в роуминге'
+export const roamingUrl = 'https://www.mts.by/help/rouming/'
```

Comment:
It's a good practice to store all URLs in one place. Let's move all URLs from separate testData files to one file. Let's name it `testData/urls`